docs/elements/hx-tile-description/index.html

```diff
@@ -0,0 +1,28 @@
+---
+title: <hx-tile-description>
+also:
+  components/choice-tiles: Choice Tiles
+  elements/hx-tile: <hx-tile>
+  elements/hx-tile-title: <hx-tile-title>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-tile-description&gt;</code> element 
+    </p>
+
+    <hx-dl class="hxBox-md metadata">
+      <hx-def>
+        <hx-dt>Permitted Parents</hx-dt>
+        <hx-dd>
+          an <a href="elements/hx-tile">&lt;hx-tile&gt;</a> element
```

Comment:
Should the link be wrapped in a set of `code` tags?

Reply:
i followed how we have the rest of the custom element's metadata set up (which are not wrapped in <code> tags, @CITguy thoughts?)

Reply:
If you've already done it this way elsewhere, then that's fine. We may have had this discussion already, actually; I'm going to blame it on Friday brain. LGTM

Reply:
Leave as-is for now. We'll revisit content strategy in future.